benchmarks/benchmark_base.py

```diff
@@ -9,11 +9,12 @@ def benchmark_cli(func):
     @click.option("-b", "--backend", type=click.Choice(["numpy", "jax"]), default="numpy")
     @click.option("-d", "--device", type=click.Choice(["cpu", "gpu"]), default="cpu")
     @click.option("-n", "--nproc", type=int, nargs=2, default=(1, 1))
+    @click.option("-ls", "--linear-solver", type=click.Choice(["petsc", "scipy", "scipy_jax", "best"]), default="best")
```

Comment:
I don't know what you mean here, I kind of need to be able to force the particular solver in command-line since I am accessing the benchmark from a bash script but maybe I shouldn't do that?

Reply:
All runtime settings are exposed as environmental variables.

```bash
$ VEROS_LINEAR_SOLVER=petsc python ...
```